.github/workflows/pr.yaml

```diff
@@ -44,7 +44,8 @@ jobs:
       needs: ${{ toJSON(needs) }}
   changed-files:
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/changed-files.yaml@branch-24.12
+    # TODO: Change back to @branch-24.12 before merging
+    uses: rapidsai/shared-workflows/.github/workflows/changed-files.yaml@fix-changed-files-too-long
```

Comment:
Reminder that this needs to be changed back before merging

Reply:
```suggestion
    uses: rapidsai/shared-workflows/.github/workflows/changed-files.yaml@branch-24.12
```

Reply:
This should not have been changed back yet. I was still in the process of testing it.

Reply:
Test was successful. https://github.com/rapidsai/shared-workflows/pull/252 has been merged. I am reverting this change again.